### ISSUE TITLE
Add Debian repository (repo.nlb.no) to system

### ISSUE THREAD

**Issue by bertfrees** [snaekobbi/system#5]
Install the reprepro-service either through Docker (but I've had some issues getting this to work on the Vagrant test VM) or using a Debian package (https://github.com/snaekobbi/reprepro-service/issues/4).

**Comment by josteinaj** [comment]
The second (Debian) approach could presumably be installed with an ansible playbook?

**Comment by bertfrees** [comment]
Yes, both approaches can be used with Ansible. But with the first I've been having some trouble. (I think if I were to test it on a real machine it would work but I use the VBox machine to test.)

**Comment by bertfrees** [comment]
I'll push the method with Docker to a branch so you can see it.

**Comment by bertfrees** [comment]
@josteinaj See commit https://github.com/snaekobbi/system/commit/82a76c757462e2e8fe217e9af61aae964b651b5e
